README.md

```diff
@@ -130,6 +130,9 @@ httpd = require('http.server').new(host, port[, { options } ])
 
   By default uses `log.info` function for requests logging.
 * `log_errors` - same as the `log_requests` option but is used for error messages logging. By default uses `log.error()` function.
+* `keepalive_disable` - disables keep-alive connections with misbehaving
+  clients. Parameter accept a map that contains a user agents with non-nil
```

Comment:
The solution with map is much easier to implement. With array, as you correctly said you need to convert to a map. Then you need to put a map to cache and correctly invalidate cache because user can change `httpd.options.keepalive_disable` in runtime.

Reply:
Added example to the README.
```patch
@@ -130,6 +130,22 @@ httpd = require('http.server').new(host, port[, { options } ])
 
   By default uses `log.info` function for requests logging.
 * `log_errors` - same as the `log_requests` option but is used for error messages logging. By default uses `log.error()` function.
+* `keepalive_disable` - disables keep-alive connections with misbehaving
+  clients. Parameter accept a map that contains a user agents with non-nil
+  value. By default map is empty.
+
+  Example:
+
+  ```lua
+  local httpd = http_server.new('127.0.0.1', 8080, {
+      log_requests = true,
+      log_errors = true,
+      keepalive_disable = {
+          ['curl/7.68.0'] = true
+      },
+  })
+  ```
+
 
 ## Using routes
 
```